PopPUNK/web.py

```diff
@@ -0,0 +1,230 @@
+import json
+import h5py
+import os 
+import re
+import sys
+import numpy as np
+import pandas as pd
+import graph_tool.topology
+import requests 
+
+def default_options():
+    """Default options for WebAPI"""
+    class args:
+        ref_db = "GPS_v3_references" 
```

Comment:
Some of this will definitely change, and would be best not put the code, and in a file instead.
What I'd suggest is to save this object as a JSON and include that in the package for now, then have this function load the JSON and parse it